File: program/program/check-lower-case/CheckLowerCase.java
import java.util.Scanner;

public class CheckLowerCase {
    public static void main(String []args) {
        Scanner my_scanner = new Scanner(System.in);
        System.out.print("Enter a character : ");
        String val = my_scanner.next();
        char ch = val.charAt(0);
        if (Character.isLowerCase(ch)) {
            System.out.println("Lower Case");
        } else {
            System.out.println("Upper Case");
        }
    }
}

File: program/program/check-lower-case/CheckLowerCase.c
#include <stdio.h> /*printf, scanf*/

int IsLowerCase(char character);

int main()
{
    char character = 0;

    int is_good = scanf("%c",&character);
    
    if(is_good != 1)
    {
        return -1;
    }

    if(IsLowerCase(character))
    {
        printf("lower case\n");
    }

    return 0;
}


int IsLowerCase(char character)
{
    if(character <= 'z' && character >= 'a')
    {
        return 1;
    }

    return 0;
}
